Snowflake.Data/Core/Authenticator/ExternalBrowserAuthenticator.cs

```diff
@@ -156,6 +157,20 @@ private static HttpListener GetHttpListener(int port)
 
         private static void StartBrowser(string url)
         {
+            string regexStr = @"^http(s?)\\:\\/\\/[0-9a-zA-Z]([-.\\w]*[0-9a-zA-Z@:])*(:(0-9)*)*(\\/?)([a-zA-Z0-9\\-\\.\\?\\,\\&\\(\\)\\/\\\\\\+&%\\$#_=@]*)?$";
```

Comment:
why we use negative regex?

Reply:
sorry, it should be !m.success, thanks for catching this. don't know why my local version is !m.success, but submit one just m.success